.gitignore

```diff
@@ -18,6 +18,7 @@ microservices/document_manager/backend/document_uploads/
 *last-backup
 venv*
 [~][$]*
+*.sln
```

Comment:
There is no need to exclude .sln files as it is a solution for the project

Reply:
We don't use Visual Studio and this was only created when I opened the project up in Visual Studio for some reason. We don't need to include it in the repo.

Reply:
it is a solution file, every time you open it in VS it will be created, and if you do not include it will be generated for every person and it might contain different configuration which is not good

Reply:
Ok, fine, I'll delete my solution file 😝